app.rb

```diff
@@ -0,0 +1,38 @@
+require 'sinatra/base'
+require './lib/users'
+require './lib/messages'
+# require './lib/security'
```

Comment:
I can see from the readme that you would have like to do more with security, I assume you've taken it out for now as it's incomplete. However when I run it, there are still functions that seem dependent on it, such as sign up